src/main/java/org/folio/processing/mapping/mapper/reader/record/marc/MarcRecordReader.java

```diff
@@ -222,30 +242,33 @@ private String retrieveNameWithoutCode(String mappingParameter) {
     return mappingParameter.substring(0, mappingParameter.trim().indexOf(FIRST_BRACKET) - 1);
   }
 
-  private void processStringExpression(MappingRule ruleExpression, boolean arrayValue, List<String> resultList, StringBuilder sb, String expressionPart) {
+  private StringBuilder processStringExpression(MappingRule ruleExpression, boolean arrayValue, List<String> resultList, StringBuilder sb, StringBuilder multipleStringBuilder, String expressionPart) {
     String value = expressionPart.replace(EXPRESSIONS_QUOTE, EMPTY);
     value = getFromAcceptedValues(ruleExpression, value);
     if (isNotEmpty(value)) {
       if (arrayValue) {
         resultList.add(value);
       } else {
         sb.append(value);
+        return new StringBuilder(value);
       }
     }
+    return new StringBuilder(EMPTY);
   }
 
-  private void processTodayExpression(StringBuilder sb) {
+  private void processTodayExpression(StringBuilder sb, StringBuilder multipleStringBuilder) {
```

Comment:
add small comment describing what this method does.

Reply:
In case when token is "###TODAY###" process this token. This method add marc today value to result

Reply:
Thanks. I meant adding the comment to the code.